src/workspace.coffee

```diff
@@ -612,16 +612,19 @@ class Workspace extends Model
 
   # Increase the editor font size by 1px.
   increaseFontSize: ->
+    @originalFontSize ?= atom.config.get("editor.fontSize")
```

Comment:
Not sure if this is worth changing things for, but this won't catch the case where someone calls `atom.config.set('editor.fontSize', 10)` directly after startup via a package or init script. In that case the "originalFontSize" would be stored as 10 even though it was a different value "originally".